File: _includes/python/success.py
import curses, sys

def success(s):
    colors = curses.tigetstr('setaf')

    def color(n):
        return curses.tparm(colors, n) if colors else ''

    sys.stdout.write(color(2) + s + color(0))

curses.setupterm()
success('Hello, World!\n')

File: _includes/c/success.c
#include <stdarg.h>
#include <stdio.h>
#include <term.h>

void success(const char *fmt, ...) {
    va_list argp;
    char *colors = tigetstr("setaf");

    if (colors)
        printf("%s", tparm(colors, 2));
    va_start(argp, fmt);
    vprintf(fmt, argp);
    va_end(argp);
    if (colors)
        printf("%s", tparm(colors, 0));
}

int main() {
    setupterm(NULL, 1, 0);
    success("Hello, World!\n");
    return 0;
}
